### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by avelanarius**
A batch of documentation fixes before the Confluent Verification:

1. Adding support information to Confluent packaging
2. Minor fixes to README
3. Adding missing LICENSE file
4. Adding a QUICKSTART guide
5. Minor cosmetic fixes to configuration parameters

One missing thing: when we release the next version of scylla-cdc-java, pom file should be updated to use the newer version.

### CODE REVIEW

**Comment by giautm** [comment]
Hi, Can you add a QUICKSTART guide with a docker-compose version?

**Comment by haaawk** [comment]
> Hi, Can you add a QUICKSTART guide with a docker-compose version?

@avelanarius would that be a lot of work?

File: README-QUICKSTART.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+```
+ALTER TABLE keyspace.table_name with cdc = {'enabled': true};
+```
+
+To learn more about Scylla CDC, visit [Change Data Capture (CDC) page](https://docs.scylladb.com/using-scylla/cdc/).
+
+## Kafka setup
+
+Scylla CDC Source Connector works well with both [open-source Kafka](https://kafka.apache.org/) 
+and [Confluent Platform](https://www.confluent.io/). In this quickstart we will show how
```
Comment (haaawk) [inline README-QUICKSTART.md:60]: Maybe note here that the version of Kafka has to be high enough?

**Review by haaawk — APPROVED** [review]
LGTM

File: README-QUICKSTART.md (hunk 1 above)
Comment (avelanarius) [reply README-QUICKSTART.md]: Added.

**Pull request merged**